pkg/xds/envoy/sockets.go

```diff
@@ -13,3 +13,7 @@ func AccessLogSocketName(name, mesh string) string {
 	}
 	return socketName[:trimLen] + ".sock"
 }
+
+func MetricsHijackerSocketName() string {
+	return "/tmp/kuma-metrics-hijacker.sock"
```

Comment:
I know it's not really a production use case, but I often run many DPs on the same (local) machine. Can we include name and mesh here to avoid conflict?